docs/taskruns.md

```diff
@@ -422,6 +422,86 @@ For more information, see the following topics:
 - For a list of supported `Volume` types, see [Specifying `VolumeSources` in `Workspaces`](workspaces.md#specifying-volumesources-in-workspaces).
 - For an end-to-end example, see [`Workspaces` in a `TaskRun`](../examples/v1beta1/taskruns/workspace.yaml).
 
+#### Propagated Workspaces
+
+**([alpha only](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features))**
+
+When using an embedded spec, workspaces from the parent `TaskRun` will be
+propagated to any inlined specs without needing to be explicitly defined. This
+allows authors to simplify specs by automatically propagating top-level
+workspaces down to other inlined resources.
+**Workspace substutions will only be made for `commands`, `args` and `script` fields of `steps`, `stepTemplates`, and `sidecars`.**
+
+```yaml
+apiVersion: tekton.dev/v1beta1 
+kind: TaskRun                  
+metadata:
+  generateName: propagating-workspaces-
+spec:
+  taskSpec:                    
+    steps:                     
+      - name: simple-step      
+        image: ubuntu          
+        command:               
+          - echo               
+        args:                  
+          - $(workspaces.tr-workspace.path)
+  workspaces:                  
+  - emptyDir: {}               
+    name: tr-workspace
+```
+
+Upon execution, the workspaces will be interpolated during resolution.
+
+```yaml
```

Comment:
Just my personal opinion but I think having the status here isn't as useful as having an english explanation of what will happen when this taskrun is run, since there's a lot of extra info in the status that isn't very relevant.

Reply:
I agree. I will hide all the non-relevant parts and provide a short description of what happens.